pandas-stubs/io/sql.pyi

```diff
@@ -1,191 +1,139 @@
 from typing import (
     Any,
-    Mapping,
-    Sequence,
+    Iterator,
+    overload,
 )
 
 from pandas.core.base import PandasObject
 from pandas.core.frame import DataFrame
 
+from pandas._typing import (
+    DtypeArg,
+    npt,
+)
+
 class DatabaseError(IOError): ...
 
-def execute(sql, con, cur=..., params=...): ...
+@overload
 def read_sql_table(
     table_name: str,
-    con,
-    schema: str | None = ...,
-    index_col: str | Sequence[str] | None = ...,
+    con: Any,
+    schema: str | list[str] = ...,
```

Comment:
`schema` can't be `list[str]` according to docs.  Also allows `None`

Reply:
Fixed.